src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/util/HttpCacheUtils.java

```diff
@@ -100,12 +102,21 @@ public static boolean responseCanSatisfyRequest(
     {
```

Comment:
`myRequestHeaders` should be renamed

Reply:
Maybe the best names for the 3 fields should be 
```
cacheResponseHeaders
cacheRequestHeaders
requestHeaders
```